plugins/auth-backend/src/service/router.ts

```diff
@@ -86,47 +86,64 @@ export async function createRouter({
     ...providerFactories,
   };
   const providersConfig = config.getConfig('auth.providers');
-  const providers = providersConfig.keys();
-
-  for (const providerId of providers) {
-    logger.info(`Configuring provider, ${providerId}`);
-    try {
-      const providerFactory = allProviderFactories[providerId];
-      if (!providerFactory) {
-        throw Error(`No auth provider available for '${providerId}'`);
+  const configuredProviders = providersConfig.keys();
+
+  for (const providerId of Object.keys(allProviderFactories)) {
```

Comment:
should probably do `for (const [providerId, providerFactory] of Object.entries(allProviderFactories))` and skip the getting + if-check of the providerFactory a few lines down

Reply:
Yes, that's obsolete now